tvm-tac/src/main/kotlin/org/ton/tac/TacInst.kt

```diff
@@ -0,0 +1,30 @@
+package org.ton.tac
+
+sealed class TacInst(
+    open val mnemonic: String
+)
+
+data class StackTacInst(
```

Comment:
I think that it should be absent in the final version of `TacInst`. But if you need it now, then ok for now.